slsDetectorGui/include/qTabDeveloper.h

```diff
@@ -20,16 +20,49 @@ class qTabDeveloper : public QWidget, private Ui::TabDeveloperObject {
   public slots:
     void Refresh();
 
+  private slots:
+    void setDetectorIndex();
+
   private:
     void SetupWidgetWindow();
     void Initialization();
     void PopulateDetectors();
-    slsDetectorDefs::dacIndex getSLSIndex(slsDetectorDefs::detectorType detType,
-                                          int index);
 
     Detector *det;
     std::vector<qDacWidget *> dacWidgets;
     std::vector<qDacWidget *> adcWidgets;
+
+    const std::vector<std::string> eiger_dacs = {
```

Comment:
Is this needed or can we get it from the API? Seems redundant but I might be missing something?

Reply:
Ah you are totally right!

Reply:
we dont have an adc list thought.. but the dacs atleast i should be able to get from the api